tests/bootstrap_project/test/project_test.dart

```diff
@@ -562,38 +588,53 @@ void main() async {
     });
   });
 
-  tearDownAll(() async {
-    try {
-      await Process.run(
-        'rm',
-        ['-rf', tempDirName],
-        workingDirectory: rootPath,
+  group('Given a created project', () {
+    final (projectName, commandRoot) = createRandomProjectName(tempPath);
+
+    late Process createProcess;
+
+    setUp(() async {
+      createProcess = await Process.start(
+        'serverpod',
+        ['create', projectName, '-v', '--no-analytics'],
+        workingDirectory: tempPath,
+        environment: {
+          'SERVERPOD_HOME': rootPath,
+        },
       );
-    } catch (e) {}
-  });
-}
+      assert((await createProcess.exitCode) == 0);
 
-(String, String) createRandomProjectName(String root) {
-  final projectName = 'test_${Uuid().v4().replaceAll('-', '_').toLowerCase()}';
-  final commandRoot = path.join(root, projectName, '${projectName}_server');
+      final docker = await Process.start(
+        'docker',
+        ['compose', 'up', '--build', '--detach'],
+        workingDirectory: commandRoot,
+      );
 
-  return (projectName, commandRoot);
-}
+      assert((await docker.exitCode) == 0);
+    });
+
+    tearDown(() async {
+      createProcess.kill();
```

Comment:
Suggestion: add a similar kill to the `docker` process.

Reply:
Maybe at the end.

Reply:
The `createProcess` variable is returned from a `start` call, whereas the `docker` one is a `run` call. So that's why this isn't needed.

Reply:
You are right, my bad here!